### DISCUSSION TITLE
Generate cover image & one-touch play from YouTube links

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#16209] [Feature Requests]
To help ignite community content for new and growing Forem's I thought it would be a good idea to explore the idea of having YouTube links auto-generate cover photos with one-touch play.

The way I see this working from a content creation perspective is like this:

Community member hits 'submit content' (or something like that.. a replacement for write post).

Member then gets an option to write a post or share a link.

'Write a post' continues as normal, 'Share a link' caters for a youtube link and title/description, once inserted the cover image is replaced with the cover image of the youtube video. The video then plays inline or it can pop out.

I feel this would be a good initial step to replacing the current video upload feature and at the same time facilitate content creation with smaller communities.

**Comment by Daxton7416** [comment]
Your idea is 100% awesome. Do you know I am also looking for such features and automation so that we could implement in our [accounting Services](http://www.accntex.com/) . And make the difficult task easy. I searched alot but haven't found any reliable source or forum which is providing such kind of automation.
